File: templates/template.vb
' BOJ 1000 C# 9.0 (.NET)
' 출처: https://www.acmicpc.net/help/language 
Module Main
 
  Sub Main()
    Dim s() As String = Nothing
 
    s = Console.ReadLine().Split(" "c)
    Console.WriteLine(CInt(s(0)) + CInt(s(1)))
  End Sub
 
End Module

File: templates/template.cs
// BOJ 1000 C# 9.0 (.NET)
//출처: https://www.acmicpc.net/help/language 
using System;

namespace Baekjoon {
    class Program {
        static void Main() {
            string s = Console.ReadLine();
            string[] ss = s.Split();
            int a = int.Parse(ss[0]);
            int b = int.Parse(ss[1]);
            Console.WriteLine(a+b);
        }
    }
}